.github/workflows/post-merge.yml

```diff
@@ -49,23 +50,14 @@ jobs:
             git add public/cards.json
             git commit -m "(CI) cards.json Generation"
           fi
-      - name: Format cards
-        if: env.SKIP_REST != 'true'
-        run: |
-          if [ ! -f .prettierrc ]; then
-            echo ".prettierrc not found"
-            echo "terminating workflow..."
-            exit 1;
-          fi
-          npx prettier public/cards.json --write
-          git add public/cards.json
-          git commit -m "(CI) Format cards" || echo "No changes to commit"
```

Comment:
The reason that this is all in one file is to avoid race conditions:

If one job pushes changes to the repo and then another job attempts later you can run into errors about merge conflicts or the branch being stale.

I think separating the jobs can be nice, so if you really wanted you could create a format job that gets triggered on `workflow_call`. But I don't think you should try to run two jobs in parallel here as it could cause one of the jobs to fail.

My personal preference here is to keep everything in one job since running prettier doesn't seem like a complex enough step that splitting it out into a separate workflow really reduces code complexity.

But I could see either approach as a valid one.

Reply:
done